test/e2e/helpers.js

```diff
@@ -43,9 +48,13 @@ async function withFixtures(options, testSuite) {
     testSpecificMock = function () {
       // do nothing.
     },
+    useBundler,
+    usePaymaster,
   } = options;
+
   const fixtureServer = new FixtureServer();
   const ganacheServer = new Ganache();
+  const bundlerServer = new Bundler();
```

Comment:
Why not just have `initBundler` return the bundler directly? It feels odd to set up `bundlerServer` here without first confirming `useBundler` is actually `true`. Keeps things cleaner and more intuitive, don't you think?

Reply:
That's technically not the server itself, but the E2E helper.

I've named it `bundlerServer` and put it there for consistency so it's defined next to the other helpers, specifically fixtures and ganache.